qgsAffine/module.py

```diff
@@ -0,0 +1,125 @@
+import pathlib
+import itertools
+
+from PyQt5 import uic
+from PyQt5.QtWidgets import QAction
+from PyQt5.QtGui import QIcon
```

Comment:
You can do `from qgis.PyQt.QtGui import X` so QGIS will automatically switch to correct PyQt version number.

Reply:
That is nice, thank you.